libs/api/domains/municipalities-financial-aid/src/lib/municipalitiesFinancialAid.service.ts

```diff
@@ -70,4 +76,34 @@ export class MunicipalitiesFinancialAidService {
       .applicationControllerGetById(applicationId)
       .catch(this.handle404)
   }
+
+  async municipalitiesFinancialAidCreateFiles(
+    auth: Auth,
+    files: ApplicationFilesInput,
+  ) {
+    return await this.fileApiWithAuth(auth)
+      .fileControllerCreateFiles({ createFilesDto: files as any })
```

Comment:
It is a bit annoying how the auto generated OpenApi client works. The client is generated from our backend where this endpoint takes in an object like this:
![image](https://user-images.githubusercontent.com/43727721/169015213-bf1f825d-0de3-4faa-994e-79baf0afb5bd.png)

But the generated object in the client becomes this: 
![image](https://user-images.githubusercontent.com/43727721/169015391-0037e2b6-b70f-4df9-b055-f19f5f03b139.png)

So I am using "any" so I can pass on a Array of objects instead of Array<string>